docs/overview/mana.md

```diff
@@ -0,0 +1,48 @@
+
+---
+## Why are fees a problem?
+There are a number of problems with fees. They create barriers to entry, can be unpredictable, and are error prone.
+
+The most obvious problem with fees is that, by definition, a fee is lost. You pay the fee and lose the value forever. This makes most blockchains "pay to play". From this respect, fee-less blockchains, like Koinos, were innevitable. We have seen this same pattern play out in the past with other technologies. Take texting, for example. When it first became available you had to pay for every text you sent. Nobody liked paying for text messages. Unlimited texting is now an expected feature of all cellular plans. In the same way, Koinos being fee-less is the future of blockchains.
+
+Other than fees not being user friendly, they are usually collected in the blockchain's native token. The native token is the "base" token of a network. On Koinos, this is KOIN. On Ethereum, it is ETH. Requiring someone to purchase a cryptocurrency before being able to use a blockchain creates a significant barrier to entry, resulting in many people being unwilling to try blockchain technology in the first place. While Mana is a property of KOIN, and requires holding KOIN to use Mana, somebody else can use their Mana to pay for you to use Koinos. Many dAPPs will pay their user's Mana so they do not even need to own KOIN before using Koinos.
+
+Fees can also be unpredictable. Fees increase when a blockchain is being used a lot and decrease when it isn't. Most of the time fees are a couple of dollars, but have spiked as high as a hundred dollars at times. Imagine sending some BTC to a friend and thinking it would only cost a few dollars but having to spend $100 instead! Because KOIN is not lost when using Koinos, even when Mana costs are flucuating (with significantly less variance than other blockchains), you are guaranteed to never lose KOIN.
+
+Lastly, using fees is error prone. Because the fee is paid for in the blockchain's native token, it can used for the fee and be transfered at the same time. There have been several instances where large fees were accidentally paid for small transactions. One person once spent [$3.1 million](https://www.coindesk.com/business/2023/11/23/bitcoin-sender-struck-with-31m-transaction-fee-largest-in-history/) on a Bitcoin transaction fee. Because Mana is not KOIN, you cannot make such mistakes on Koinos. Your KOIN is always safe!
+
+---
+## What is Mana?
+Now that we know why fees are a problem and how Mana fixes those problems, we can talk about what Mana is exactly.
+
+Mana is a property of a KOIN token. Every single KOIN can have an equal amount of Mana. If you have 1 KOIN, you also have up to 1 Mana. Likewise, if you have 10 KOIN, you also have up to 10 Mana. Sometimes it is helpful to think of Mana as an absolute value (10 Mana) and other times it is helpful to think of it as a percent (100%). If you have 10 KOIN and your Mana is at 50%, you have 5 Mana and vis versa.
```

Comment:
vis versa might be confusing here.

Reply:
Also, I think it might be good to emphasize that Mana is not a token. Everyone I explain Mana to immediately thinks Mana is a token even after I tell them that Mana is a PROPERTY of KOIN.